db/schema.rb

```diff
@@ -39,6 +39,16 @@
     t.index ["singleton_guard"], name: "index_myconfigs_on_singleton_guard", unique: true
   end
 
+  create_table "offsprings", force: :cascade do |t|
+    t.string   "first_name", null: false
+    t.string   "last_name", null: false
+    t.string   "type", null: false
```

Comment:
Youre right, its obvious. But maybe I did not express myself properly. Because if you have a Single Table, with fields age & grade, if you create an AgedOffspring, shouln't the grade value be `grade= null`. ? so, type will have to be as it is.?